src/domains/queries.ml

```diff
@@ -9,6 +9,9 @@ module VDQ = ValueDomainQueries
 module ID = VDQ.ID
 
 module LS = VDQ.LS
+
+module AD = Lattice.LiftTop (ValueDomain.AD)
```

Comment:
Why does this need a `LiftTop`? `AD` already has a top.

Reply:
If it is not lifted, Goblint crashes with `Goblint_lib.Lattice.Unsupported("partial map top")`:

```
Fatal error: exception Goblint_lib.Lattice.Unsupported("partial map top")
Raised at Goblint_lib__Lattice.unsupported in file "src/domains/lattice.ml", line 49, characters 20-41
Called from Goblint_lib__MCP.MCP2.query' in file "src/analyses/mCP.ml", line 312, characters 37-52
Called from Goblint_lib__Analyses.ask_of_ctx.(fun) in file "src/framework/analyses.ml" (inlined), line 352, characters 81-90
Called from Goblint_lib__Written.Spec.assign in file "src/analyses/written.ml", line 26, characters 10-39
Called from Goblint_lib__MCP.MCP2.assign.f in file "src/analyses/mCP.ml", line 379, characters 17-34
Called from Goblint_lib__MCP.MCP2.map_deadcode.one_el in file "src/analyses/mCP.ml", line 100, characters 49-78
Called from Stdlib__List.fold_left in file "list.ml", line 121, characters 24-34
Called from Goblint_lib__MCP.MCP2.map_deadcode in file "src/analyses/mCP.ml", line 101, characters 13-35
Called from Goblint_lib__MCP.MCP2.assign in file "src/analyses/mCP.ml", line 381, characters 15-52
Called from Goblint_lib__Constraints.WidenContextLifterSide.lift_fun in file "src/framework/constraints.ml", line 373, characters 25-41
```

Reply:
Apparently there's this TODO:
https://github.com/goblint/analyzer/blob/17348bec5283822b54b6dd4503a8bb7b617a15ed/src/cdomains/addressDomain.ml#L146-L147